model/src/cfd.rs

```diff
@@ -913,14 +967,25 @@ impl Cfd {
         }
 
         let now = OffsetDateTime::now_utc();
-
         let candidate_event_id = olivia::next_announcement_after(now + self.settlement_interval);
+
+        let from_event_id = if let Some(from_event_id) = from_event_id {
+            from_event_id
+        } else {
```

Comment:
Note: I added the statement when we `start` the rollover because I think it makes more sense to log the information there. 
At this point in time we cannot say for sure if it is a retry, because the `start_rollover_maker` logic sets the `from_event_id` also when it is the current one. I implemented it this way because I think it makes the feature more complete. The fact that we have to evaluate if `from_event_id` is `Some` in `accept_rollover_proposal` is just due to being backwards compatible. 

In the future we can opt for making the `settlement_event_id` always set in `RevokeCommit`, i.e. remove the `Option` and then this check for `Some` in `accept_rollover_proposal` would go away.